packages/example/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@navigation-ex/example",
+  "version": "0.0.0",
+  "private": true,
+  "workspaces": {
```

Comment:
workspaces is supposed to be present only in the root of the repo, isn't it?

Reply:
Yarn docs says we can use this to configure hoisting for specific packages. For the expo app, I just disabled hoisting since it causes issues with RN.

Reply:
Ok makes sense